packages/commerce-sdk-react/src/hooks/useDNT.ts

```diff
@@ -20,33 +21,36 @@ interface useDntReturn {
  * @returns {Object} - The returned object containing DNT states and function to update preference
  * @property {boolean | undefined} dntStatus @deprecated - DNT user preference. Used to determine
  *              if the consent tracking form should be rendered
- * **Deprecated since version 3.1.0 Use selectedDoNotTrackValue instead.**
- * @property {boolean} selectedDoNotTrackValue - DNT user preference. Used to determine
+ * **Deprecated since version 3.1.0 Use selectedDnt instead.**
+ * @property {boolean} selectedDnt - DNT user preference. Used to determine
  *              if the consent tracking form should be rendered
- * @property {boolean} effectiveDoNotTrackValue - effective DNT value to apply to
- *              analytics layers. Takes defaultDnt into account when selectedDoNotTrackValue is undefined.
+ * @property {boolean} effectiveDnt - effective DNT value to apply to
+ *              analytics layers. Takes defaultDnt into account when selectedDnt is undefined.
  *              If defaultDnt is undefined as well, then SDK default is used.
  * @property {function} updateDNT - takes a DNT choice and creates the dw_dnt
  *              cookie and reauthorizes with SLAS
  *
  */
 const useDNT = (): useDntReturn => {
     const auth = useAuthContext()
-    const selectedDoNotTrackValue = auth.getDnt()
-    const effectiveDoNotTrackValue = auth.getDnt({
+    const selectedDnt = auth.getDnt()
+    const effectiveDnt = auth.getDnt({
         includeDefaults: true
     })
     const updateDNT = async (preference: boolean | null) => {
         await auth.setDnt(preference)
     }
-    const dntStatus = selectedDoNotTrackValue
+    const updateDnt = updateDNT
+    const dntStatus = selectedDnt
 
     return {
-        selectedDoNotTrackValue,
-        effectiveDoNotTrackValue,
-        /** @deprecated - Deprecated since version 3.1.0. Use selectedDoNotTrackValue instead. */
+        selectedDnt,
+        effectiveDnt,
+        /** @deprecated - Deprecated since version 3.1.0. Use selectedDnt instead. */
```

Comment:
Please move this block to jsdoc comment so it will be generated into the typedoc when we merge.